lib/class-wp-theme-json-schema-v0.php

```diff
@@ -260,12 +258,33 @@ private static function process_settings( $settings, $blocks_to_consolidate ) {
 	 * Processes the styles subtree.
 	 *
 	 * @param array $styles Array to process.
-	 * @param array $blocks_to_consolidate A list of blocks that are transformed.
 	 *
 	 * @return array The styles in the new format.
 	 */
-	private static function process_styles( $styles, $blocks_to_consolidate ) {
-		$new = array();
+	private static function process_styles( $styles ) {
+		$new                   = array();
+		$block_heading         = array(
```

Comment:
This code is to map a previous shape to a new one right? It will not need to be part of core? I guess if so we can have a comment saying not port this code to core.

Reply:
Added a comment in the WP_Theme_JSON constructor.